setup.py

```diff
@@ -31,6 +31,9 @@
     keywords='pydocstyle, PEP 257, pep257, PEP 8, pep8, docstrings',
     packages=('pydocstyle',),
     package_dir={'': 'src'},
+    install_requires=[
+        'jedi==0.9.0',
```

Comment:
Please, let's not pin this. This causes unending headaches for users and downstream redistributors.